app/src/main/java/de/westnordost/streetcomplete/settings/SettingsFragment.kt

```diff
@@ -111,9 +115,13 @@ class SettingsFragment : PreferenceFragmentCompat(),
             }
             Prefs.AUTOSYNC -> {
                 if (Prefs.Autosync.valueOf(prefs.getString(Prefs.AUTOSYNC, "ON")!!) != Prefs.Autosync.ON) {
+                    val view = LayoutInflater.from(activity).inflate(R.layout.dialog_tutorial_upload, null)
+                    val filled = context!!.getString(R.string.action_download)
+                    val uploadExplanation = view.findViewById<MaterialTextView>(R.id.tutorialDownloadPanel)
```

Comment:
MaterialTextView? It think it is a normal text view

Reply:
Weird, I added it to avoid crash. Removed.